app/model/asana.js

```diff
@@ -207,16 +207,27 @@ const Asana = {
         }
         return false
     },
-    async loadSections() {
-        if (jsonstore.has('sections')) {
+    async loadSections(withCache) {
+        this.clearSectionsData()
+        if (jsonstore.has('sections') && withCache) {
             let response = jsonstore.get('sections')
             this.processSections(response)
-        }
-        x.request({ url: `https://app.asana.com/api/1.0/projects/${this.projectId}/sections` }).then((response) => {
+        } 
 
+        await x.request({ url: `https://app.asana.com/api/1.0/projects/${this.projectId}/sections` }).then((response) => {
+            
             jsonstore.set('sections', response)
             this.processSections(response)
         })
+        
+    },
+    clearSectionsData() {
```

Comment:
I'd move all the clear*() functions to the top so they're close to the variable definitions so if someone else adds some new variables, they'll see that they should also consider clearing them.